conformance/utils/kubernetes/helpers.go

```diff
@@ -626,8 +627,25 @@ func listenersMatch(t *testing.T, expected, actual []v1beta1.ListenerStatus) boo
 			t.Logf("Name doesn't match")
 			return false
 		}
-		if !reflect.DeepEqual(aListener.SupportedKinds, eListener.SupportedKinds) {
-			t.Logf("Expected SupportedKinds to be %v, got %v", eListener.SupportedKinds, aListener.SupportedKinds)
+
+		if len(eListener.SupportedKinds) == 0 && len(aListener.SupportedKinds) != 0 {
+			t.Logf("Expected list of SupportedKinds was empty, but the actual list for comparison was not:  %v",
+				aListener.SupportedKinds)
+			return false
+		}
+		// Ensure that the expected Listener.SupportedKinds items are present in actual Listener.SupportedKinds
+		// Find the items instead of performing an exact match of the slice because the implementation
+		// might support more Kinds than defined in the test
+		eSupportedKindsSet := sets.NewString()
+		aSupportedKindsSet := sets.NewString()
+		for _, eKind := range eListener.SupportedKinds {
+			eSupportedKindsSet.Insert(string(eKind.Kind))
+		}
+		for _, aKind := range aListener.SupportedKinds {
+			aSupportedKindsSet.Insert(string(aKind.Kind))
+		}
```

Comment:
With the latest utils you should be able to do this, right?

```suggestion
		eSupportedKindsSet := sets.New(eListener.SupportedKinds...)
		aSupportedKindsSet := sets.New(aListener.SupportedKinds...)
```